org.freeciv.qt.yml

```diff
@@ -0,0 +1,25 @@
+app-id: org.freeciv.qt
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: freeciv-qt
+rename-icon: freeciv-client
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.freeciv:create
+modules:
+  - name: freeciv-qt
+    buildsystem: simple
```

Comment:
you should be able to do that with buildsystem `autotools`

Reply:
Yeah, this should really be changed to use `autotools`.